chaos_kitten/cli.py

```
@@ -175,371 +174,84 @@ def scan(
-
-    summary = diff_result["summary"]
-    critical_findings = diff_result["critical_findings"]
-
-    # Display diff summary
-    console.print()
-    console.print(Panel(
-        f"""[bold]Diff Summary:[/bold]
+            from chaos_kitten.utils.config import Config
+            config_obj = Config(config)
+            cfg = config_obj.load()
+        except FileNotFoundError as e:
```

Ruff (0.15.1)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)